curriculum/challenges/english/25-front-end-development/quiz-semantic-html/66ed903cf45ce3ece4053ebe.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is presentational HTML?
 
```

Comment:
HTML5, HyperText Markup Language, is code created that is HTML5 and has stylistic value.